projects/storefrontapp-e2e-cypress/cypress/helpers/homepage.ts

```diff
@@ -1,7 +1,13 @@
-export function checkBanner() {
-  cy.get(
-    'cx-page-slot cx-banner img[alt="Save Big On Select SLR & DSLR Cameras"]'
-  ).should('exist');
+export function checkBanner(isPowertools = false) {
+  if (!isPowertools) {
```

Comment:
How about apparel?

Reply:
it's just a variable, but yeah I will change it to make it more meaningful
maybe something like isOtherBaseSite